addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3245,6 +3245,10 @@ export class OdooEditor extends EventTarget {
         for (const el of element.querySelectorAll('[contenteditable="false"]')) {
             el.setAttribute('data-oe-keep-contenteditable', '');
         }
+        // Flag elements .oe-tabs contenteditable=false.
+        for (const el of element.querySelectorAll('.oe-tabs')) {
+            el.setAttribute('contenteditable', 'false');
```

Comment:
Why is this needed ? The only place where we create `oe-tabs` elements they already have `contentEditable=false`.

Reply:
We clean `contenteditable=false` during save (OdooEditor:3275) : 
```javascript 
// Remove contenteditable=false on elements
for (const el of element.querySelectorAll('[contenteditable="false"]')) {
    if (!el.hasAttribute('data-oe-keep-contenteditable')) {
        el.removeAttribute('contenteditable');
    }
}
```

So we either need to add it back when start the edition process ( current version)
Or we need to flag the `.oe-tabs` with the `data-oe-keep-contenteditable` property.

The first solution seems the best to me, because we don't really want to keep a bunch of `contenteditable=false` span out of the edition process. Seems a bit dirty IMO.

Though ?